swagger.yaml

```diff
@@ -1,42 +1,292 @@
+        description: |
+          The state of the image, can only be one of the following:
+          * created
+          * published
+        example: "published"
+      filename:
         type: string
-        description: "Message returned by hello world endpoint"
-        example: "Hello, world!"
\ No newline at end of file
+        description: "image's file name"
+        example: "imageName.jpg"
+      license:
+        type: object
+        description: "license that defines the third party permissions for this image"
+        properties:
+          title:
+            type: string
+            description: "Title of the license"
+            example: "Open Government Licence v3.0"
+          href: 
+            type: string
+            description: "Link to the license content"
+            example: "https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+      original:
+        type: object
+        description: "original file upload in the upload bucket"
+        properties:
+          path:
+            type: string
+            description: "S3 object key (without bucket name) where the image is stored"
+            example: "images/042e216a-7822-4fa0-a3d6-e3f5248ffc35/imageName.jpg"
+      type:
+        type: string
+        description: "type of image, which might define a set of possible resolutions."
+        example: "chart"
+      downloads:
+        type: object
+        description: "all available download variants for this image, keyed by resolutions (in WxH pixels) and/or formats. Note that this spec does not define all possible resolutions, and depending on the image type, we can have multiple formats as well as multiple resolutions."
+        properties:
+          1920x1080:
+            $ref: '#/definitions/ImageDownload'
+          1280x720:
+            $ref: '#/definitions/ImageDownload'
+          1920x1080_png:
+            $ref: '#/definitions/ImageDownload'
+          1920x1080_jpg:
+            $ref: '#/definitions/ImageDownload'
+                      
+          
+  ImageDownload:
+    type: object
+    description: "Download information for a particular image and resolution"
+    properties:
+      size:
+        type: integer
+        description: "File size in number of bytes"
+        example: 1024
+      href:
+        type: string
+        description: "Full URL pointing to the image in download service"
+        example: "http://dp-download-service/images/042e216a-7822-4fa0-a3d6-e3f5248ffc35/imageName.jpg"
+      public:
+        type: string
+        description: "S3 Public bucket name"
+        example: "my-pubic-bucket"
+      private:
+        type: string
+        description: "S3 Private bucket name"
+        example: "my-private-bucket"
+
+securityDefinitions:
+
+  FlorenceAPIKey:
+    name: florence-token
+    description: "API key used to allow florence users to create and query images metadata"
+    in: header
+    type: apiKey
+
+  InternalAPIKey:
```

Comment:
This header has been deprecated and is no longer in use so we won't need it on this API.

Reply:
Originally, I copied these vaues from dataset api swagger. But I just double-checked the actual names of the headers, and they should are:
```
Florence -> X-Florence-Token
Service ->Authorization
```
I will use these values.